Makefile

```diff
@@ -97,7 +97,7 @@ report: ## publish test coverage report
 	$(COMPOSE_RUN) -e CODECOV_TOKEN lms codecov --commit=${CIRCLE_SHA1}
 .PHONY: report
 
-run: ## start lms development server and nginx
+run: tree ## start lms development server and nginx
```

Comment:
:thinking: you should remove the `tree` dependency from the `bootstrap` rule then. And commit this change apart as it's not related with the bug fix.

Reply:
how shall i describe this modification ? it was working before, and only fails on the CI